### DISCUSSION TITLE
Do you plan on updating this API for Tears of the Kingdom

### DISCUSSION THREAD

**Discussion by arjunkahlon** [gadhagod/Hyrule-Compendium-API#36] [General]
I've really enjoyed using this API to create my own personal project. I envision the new game having a variety of new items, creatures, monsters. If Nintendo does opt to continue the Compendium, will your team continue to update this API with the new data?

**Comment by gadhagod** [comment]
This is a good question! I think we will have both games supported. You will probably be able to configure which game you're asking about with a parameter, for example:

    https://botw-compendium.herokuapp.com/api/v2/entry/bokoblin?game=2 // 2 means the sequel, 1 means the original

The API will likely point to the original game by default. 

I'm excited about the sequel!

**Comment by erikmon** [comment]
Hi @gadhagod, any news? It would be a good idea.

**Reply by gadhagod** [reply]
Thanks for reminding me! About to start working on it. Will try to get the new features out ASAP.

Edit: see https://github.com/gadhagod/Hyrule-Compendium-API/pull/40.

**Comment by gadhagod** [comment]
@arjunkahlon @erikmon it's here! Check out the [new docs](https://gadhagod.github.io/Hyrule-Compendium-API/#/compendium-api?id=tears-of-the-kingdom-entries) to see how to access Tears of the Kingdom entries. I deployed the new features just a few hours ago so please let me know if you come across any issues.

TOTK images and regions aren't ready yet; they are still being tested.

**Reply by erikmon** [reply]
Thank you so much for your support @gadhagod

**Reply by arjunkahlon** [reply]
Thank you @gadhagod! I'm really excited to update my project to toggle between BOTW and TOTK.

**Reply by gadhagod** [reply]
Glad I could help guys!

**Reply by jkingster** [reply]
Any time estimate of when the regions will be released? And are you looking for any contributions? I'd be willing to help in any matter regarding this. I see you just released the TOTK entries, so my apologies for arrogance. I don't want to make it seem like I'm rushing you. Just curious.

**Reply by gadhagod** [reply]
Thanks for your message, @JacobDevelopment. I am pretty busy this summer due to my internship, so it may be a few weeks before regions can be released. However, I'm glad to hear that you're open to contributing. The local hosting instructions are [here](https://gadhagod.github.io/Hyrule-Compendium-API/#/self-hosting). Since there is no real contributing guide or anything, I can walk you through the codebase if you're open to communicating directly with me at good.aarav@gmail.com or "gadhagod" on Discord.

**Reply by jkingster** [reply]
No problem, I'll look to getting this setup and contact you via discord as soon as I can.

**Reply by ciaracade** [reply]
Was looking for a BOTW api that hopefully had updates with TOTK content as well. Glad I found this!
